subsys/shell/Kconfig.backends

```diff
@@ -24,6 +24,12 @@ config SHELL_BACKEND_SERIAL
 
 if SHELL_BACKEND_SERIAL
 
+config SHELL_PROMPT_UART
+	string "Displayed prompt name."
```

Comment:
Nit: Usually not `.` at the end of a prompt.

Reply:
ok, will remove

Reply:
done